tests/modules/create.py

```diff
@@ -1,55 +1,65 @@
 import os
 
 import pytest
-import requests_mock
+import requests_cache
+import responses
 
 import nf_core.modules
-from tests.utils import mock_api_calls
+from tests.utils import mock_anaconda_api_calls, mock_biocontainers_api_calls
 
 
 def test_modules_create_succeed(self):
     """Succeed at creating the TrimGalore! module"""
-    with requests_mock.Mocker() as mock:
-        mock_api_calls(mock, "trim-galore", "0.6.7")
+    with responses.RequestsMock() as rsps:
+        mock_anaconda_api_calls(rsps, "trim-galore", "0.6.7")
+        mock_biocontainers_api_calls(rsps, "trim-galore", "0.6.7")
         module_create = nf_core.modules.ModuleCreate(
             self.pipeline_dir, "trimgalore", "@author", "process_single", True, True, conda_name="trim-galore"
         )
-        module_create.create()
+        with requests_cache.disabled():
```

Comment:
Which cache is this affecting? Wondering if it ignores the cloning of the modules repository and then we have to clone it again for every test, but it doesn't seem so when I try to run the test :)

Reply:
No, it's not the creation of that, it's just any queries that are running inside that code. For example, if you try to hit anaconda here. It was masking the mock and what I was stuck on months ago because the "mock wasn't used" because it was using the cache.